app/controllers/chargeback_controller.rb

```diff
@@ -237,17 +277,28 @@ def cb_rate_edit
             fixture.each do |cbr|
               if cbr[:rate_type] == x_node.split('-').last
                 rates = cbr.delete(:rates)
-                rates.each do |r|
+                rates.each_with_index do |r, i|
                   detail = ChargebackRateDetail.new
+                  @sb[:tiers][i] = []
                   detail.description = r[:description]
                   detail.source = r[:source]
                   # detail.rate = r[:rate]
                   # detail.per_time = r[:per_time]
-                  detail.rate = ""
                   detail.per_time = "hourly"
                   detail.group = r[:group]
                   detail.per_unit = r[:per_unit]
                   detail.metric = r[:metric]
+                  tiers = r.delete(:tiers)
+                  rate_tiers = []
+                  tiers.each do |t|
+                    tier = ChargebackTier.new
+                    tier.start = t.delete(:start)
+                    tier.end = t.delete(:end)
+                    tier.fix_rate = t.delete(:fix_rate)
+                    tier.var_rate = t.delete(:var_rate)
+                    tier.chargeback_rate_detail_id = detail.id
+                    rate_tiers.append(tier)
```

Comment:
Same as above, let's pass the attributes to the model with the .new or .create call